tasks/update-granules-metadata/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@cumulus/update-granules-metadata",
+  "version": "2.0.1",
+  "description": "Update cmr metadata files with correct online access urls and etags and transfer etag info to granules' CMR files",
+  "main": "index.js",
+  "directories": {
+    "test": "tests"
+  },
+  "homepage": "https://github.com/nasa/cumulus/tree/master/tasks/update-granules-metadata",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nasa/cumulus"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "engines": {
+    "node": ">=12.18.0"
+  },
+  "scripts": {
+    "build": "rm -rf dist && mkdir dist && cp -R schemas dist/ && ../../node_modules/.bin/webpack",
+    "package": "npm run build && (cd dist && node ../../../bin/zip.js lambda.zip index.js schemas)",
+    "test": "../../node_modules/.bin/ava",
+    "test:coverage": "../../node_modules/.bin/nyc npm test",
+    "watch": "rm -rf dist && mkdir dist && cp -R schemas dist/ && ../../node_modules/.bin/webpack --progress -w"
+  },
+  "ava": {
+    "files": [
+      "tests/*.js"
+    ],
+    "fail-fast": true,
+    "serial": true,
+    "verbose": true,
+    "timeout": "15m"
+  },
+  "author": "Cumulus Authors",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@cumulus/aws-client": "2.0.1",
+    "@cumulus/cmrjs": "2.0.1",
+    "@cumulus/common": "2.0.1",
+    "@cumulus/cumulus-message-adapter-js": "1.3.1",
+    "@cumulus/errors": "2.0.1",
+    "@cumulus/ingest": "2.0.1",
```

Comment:
I don't think either of these packages are used in the code for this task that I can tell?